spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +96,52 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+@pytest.mark.parametrize('path_method', ['absolute', 'relative'])
+def test_copy_path(explorer_with_files, path_method):
+    """Test copy absolute and relative paths."""
+    project, _, file_paths, cb = explorer_with_files
+    project.explorer.treewidget.copy_path(fnames=file_paths,
+                                          method=path_method)
+    cb_output = cb.text(mode=cb.Clipboard)
+    path_list = [path.strip(',"') for path in cb_output.splitlines()]
+    for path in path_list:
```

Comment:
You need to  ``assert len(path_list) == file_paths`` before doing this, otherwise if e.g. only one path is actually stored in the clipboard, or even none at all (i.e. the function fails completely) the test will still pass.

Furthermore, you should just do ``for expected_path, actual_path in zip(file_paths, path_list) and check if the two paths are equal (after ``osp.normpath()``), since its a much cleaner, and clearer and more appropriate check.